bundle/API/Repository/TagsService.php

```diff
@@ -286,6 +286,22 @@ public function newSynonymCreateStruct(int $mainTagId, string $mainLanguageCode)
      */
     public function newTagUpdateStruct(): TagUpdateStruct;
 
+    /**
+     * Hides $tag.
+     *
+     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\UnauthorizedException If the current user is not allowed to delete this tag
+     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\NotFoundException If the specified tag is not found
+     */
+    public function hideTag(Tag $tag): void;
+
+    /**
+     * Unhides $tag.
+     *
+     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\UnauthorizedException If the current user is not allowed to delete this tag
+     * @throws \Ibexa\Contracts\Core\Repository\Exceptions\NotFoundException If the specified tag is not found
+     */
+    public function unhideTag(Tag $tag): void;
```

Comment:
I would go with `revealTag()` here, to be inline with the newer implementation in Repository API and with labeling in the Admin UI. (We do have `LocationService::unhideLocation()`, but corresponding newer implementation in `ContentService` is named `revealContent()`.